frotz.py

```diff
@@ -187,30 +211,54 @@ def _paste_cb(self, button):
         self._vte.paste_clipboard()
 
     def __key_press_cb(self, window, event):
-        if event.state & gtk.gdk.CONTROL_MASK and event.state & gtk.gdk.SHIFT_MASK:
+        if event.state & Gdk.ModifierType.CONTROL_MASK and event.state & Gdk.ModifierType.SHIFT_MASK:
         
-            if gtk.gdk.keyval_name(event.keyval) == "C":
+            if Gdk.keyval_name(event.keyval) == "C":
                 if self._vte.get_has_selection():
                     self._vte.copy_clipboard()              
                 return True
-            elif gtk.gdk.keyval_name(event.keyval) == "V":
+            elif Gdk.keyval_name(event.keyval) == "V":
                 self._vte.paste_clipboard()
                 return True
-                
+
         return False
 
-class VTE(vte.Terminal):
-    def __init__(self):
-        vte.Terminal.__init__(self)
-        self._configure_vte()
 
-        #os.chdir(os.environ["HOME"])
-        self.fork_command()
+class VTE(Vte.Terminal):
 
-    def _configure_vte(self):
-        conf = ConfigParser.ConfigParser()
+    def __init__(self):
+        Vte.Terminal.__init__(self)
+        self.configure_terminal()
+
+        os.chdir(os.environ["HOME"])
```

Comment:
can you provide an inline comment explaining why this was uncommented or why this is necessary?

Reply:
Thanks for noticing, The line seems to have no effect. Actually activity was crashing before, so I copied some of the code from VncLauncher activity. I think it is good to remove line, if it has no effect.

Reply:
Great